lowstepper_wasm_demo/src/App.tsx

```diff
@@ -180,39 +216,323 @@ export default function App() {
+              setState({ ...state, "gate": true })
+            }}
+            onMouseUp={(e) => {
+              setState({ ...state, "gate": false })
+            }}
+            onMouseOut={() => {
+              setState({ ...state, "gate": false })
+            }}>
+            </button>
+        </div>
+
+        <div className="buttonWrapper white third_row">
+            <button className="button" onMouseDown={(e) =>  {
+              setState({ ...state, "gate": true })
+            }}
+            onMouseUp={(e) => {
+              setState({ ...state, "gate": false })
+            }}
+            onMouseOut={() => {
+              setState({ ...state, "gate": false })
+            }}>
+            </button>
+            <h3 className="parameterName outsTitle">OUTS</h3>
+        </div>
+
+        <div className="buttonWrapper white">
+            <button className="button" onMouseDown={(e) =>  {
+              setState({ ...state, "gate": true })
+            }}
+            onMouseUp={(e) => {
+              setState({ ...state, "gate": false })
+            }}
+            onMouseOut={() => {
+              setState({ ...state, "gate": false })
+            }}>
+            </button>
+        </div>
+
+      </div>
+      <div className="container trigBigWrapper">
+        <div className="trigWrapper">
+          
+          <div className="buttonWrapper red">
+            <button className="button" onMouseDown={(e) =>  {
+              setState({ ...state, "gate": true })
+            }}
+            onMouseUp={(e) => {
+              setState({ ...state, "gate": false })
+            }}
+            onMouseOut={() => {
+              setState({ ...state, "gate": false })
+            }}>
+            </button>
+          </div>
+
+          <h3 className="parameterName">TRIG <img src="static/images/ico_trig.png" /></h3>
 
-      <div className="container">
-        <div>
-          <h2 className="parameterName">Trig</h2>
-          <p className="parameterDescription">Click here to trigger the LFO.</p>
-          <button onMouseDown={(e) =>  {
-            setState({ ...state, "gate": true })
-          }}
-          onMouseUp={(e) => {
-            setState({ ...state, "gate": false })
-          }}
-          onMouseOut={() => {
-            setState({ ...state, "gate": false })
-          }}>
-            Trig
-          </button>
-
-          <button onClick={() => {
+          <button className="autoTrigBtn" onClick={() => {
             if(trigHandler.autoTrigHandler) {
               trigHandler.stopAutoTriggerA();
               return;
             }
 
             trigHandler.startAutoTriggerA(1000);
-          }}>TOGGLE AUTO TRIG (fixed 1000ms)</button>
+          }}>AUTO TRIG (1000ms)</button>
+
+        </div>
+        <div className="trigWrapper">
+          <div>
+              <div className="buttonWrapper red">
+                <button className="button" onMouseDown={(e) =>  {
```

Comment:
wrap `setState({ ...state, "gate": true })` in some function

you can use a react component as well for the repetition of onMouseUp / onMouseOut, etc